build.mill

```diff
@@ -1,3 +1,4 @@
+//> mill-version =
```

Comment:
@lihaoyi Was this line addition intentional?

Reply:
@lefou oh probably not, must have committed it accidentally. Let me remove it